hash.bat

```diff
@@ -0,0 +1,67 @@
+:: hash.bat
```

Comment:
So this is strictly for the GitHub release flow, or does it serve other purposes too?

Can you please edit the PR description with context, any relevant issues, etc.?

Reply:
Sorry this is still WIP -- I'm not sure exactly how to get this done.

It is just for github, but it doesn't work when I place hash.bat in the .github/ folder. Some Windows permission issue that I haven't investigated deeply yet.